docker/docker-compose.yml

```
@@ -128,7 +128,7 @@ services:
   frontend:
     healthcheck:
       test: ["CMD", "curl", "-f", "http://localhost:8080/actuator/health"]
-    image: quay.io/ukhomeofficedigital/hocs-frontend
+    image: quay.io/ukhomeofficedigital/hocs-frontend:${FRONTEND_TAG}
```

You can use shell expansion here!
That way you'll only need to put non-mainline versions into your `.env`.
```suggestion
    image: quay.io/ukhomeofficedigital/hocs-frontend:${FRONTEND_TAG:-latest}
```

---

Ohhh I didn't realise that, that's cool. However by doing that how do we manage both sets of envvars in the `.env` file, just comment out the ones we don't need at the time or something?